src/views/QuestionBank/js/question-bank-form.js

```diff
@@ -0,0 +1,117 @@
+import { mapActions, mapGetters } from 'vuex'
+import Editor from '@/components/editor/Editor'
+
+export default {
+  name: 'QuestionBankForm',
+  props: [
+    'editMode'
+  ],
+  components: {
+    Editor
+  },
+  data () {
+    return {
+      questionBankDetail: {
+        id: '',
+        title: '',
+        description: ''
+      },
+      isSubmitting: false
+    }
+  },
+  created() {
+    this.initPage()
+  },
+  computed: {
+    ...mapGetters([
+      'questionBank'
+    ])
+  },
+  methods: {
+    ...mapActions([
+      'fetchQuestionBankDetail',
+      'createQuestionBank',
+      'updateQuestionBank'
+    ]),
+    initPage() {
+      if (!!this.editMode) {
```

Comment:
same as this `!!`

Reply:
Yes, !! is used to handle truthy falsy value in javascript. CMIIW